lib/airbrake.js

```diff
@@ -59,6 +59,11 @@ Airbrake.createClient = function(projectId, key, env) {
   instance.key = key;
   instance.env = env || instance.env;
   instance.projectId = projectId || instance.projectId;
+
+  if (!instance.key) {
```

Comment:
Ok, I can take care of that.  I'd seen multiple tests leave out the `projectId` field so I assumed that the project key wasn't 100% required (though in hindsight the key _is_ required, so we might as well add checks for the `projectId` as well).  Thanks for reviewing!